test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -270,6 +270,7 @@ namespace integration_framework {
   IntegrationTestFramework &IntegrationTestFramework::setGenesisBlock(
       const shared_model::interface::Block &block) {
     iroha_instance_->makeGenesis(clone(block));
+    iroha_instance_->init();
```

Comment:
Not sure that the new call is required here.

Reply:
Discussed. Briefly, the reason is in this change:
![Screenshot_2019-05-20 ITF fixed init failure handling by MBoldyrev · Pull Request #70 · hyperledger iroha](https://user-images.githubusercontent.com/11086641/58071931-cc118a80-7ba6-11e9-8a42-542a22e3ffe1.png)
So previously the `init` call was made implicitly inside the `makeGenesis` method, which is not a good design as for me. Now it is converted into an explicit separate method.